blog/siglens-1025x-faster-than-elasticsearch.html

```diff
@@ -0,0 +1,1075 @@
+                                <pre class="language-javascript"><code class="language-javascript">curl -k -u "elastic:" \
+    --location \
+    --request PUT 'https://<your_elasticsearch_host>:9200/_template/temp1' \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+        "index_patterns": "trips",
+        "settings": {
+            "number_of_shards": 6,
+            "number_of_replicas": 0
+        },
+        "mappings": {
+            "_source": {
+                "enabled": true
+            },
+            "properties": {
+                "timestamp": {
+                    "type": "date",
+                    "format": "epoch_millis"
+                }
+            }
+        }
+    }'</code></pre>
+                            </div>
+                        </div>
+                        <div>
+                            <h2 class="sub-heading mt-2 section" id="section4-4">Save the password for later
+                            </h2>
+                            If you don't know the password for the elastic user, reset it with
+                            <pre
+                                class="language-javascript"><code class="language-javascript">bin/elasticsearch-reset-password -u elastic</code></pre>The new password will be printed. Save it for later.
+                        </div>
+                        <div>
+                            <h2 class="sub-heading mt-2 section" id="section4-5">Run Elasticsearch
+                            </h2>
+                            <pre
+                                class="language-javascript"><code class="language-javascript">./bin/elasticsearch > elastic.log 2>&1</code></pre>
+                        </div>
+                        <div>
+                            <h2 class="sub-heading mt-2 section" id="section4-6">Ingest the Data</h2>In Terminal 2, clone siglens to use an ingester script from it.
+                            <pre
+                                class="language-javascript"><code class="language-javascript">cd /mnt nvme1n1
+git clone https://github.com/siglens/siglens.git
+cd siglens/tools/nyc-taxi-benchmark</code></pre>
+                                    In ingester.py, make these changes:
+                            <ul>
+                                <li>Change <span class="courier-font">index_data = '{"index": {"_index": "trips", "_type": "_doc"}}' to index_data = '{"index": {"_index": "trips"}}'</span></li>
+                                <li>In the curl section, add the <span class="courier-font">"-k"</span>, option, which allows faking SSL checks</li>
+                                <li>In the curl section, add <span class="courier-font">"-u", "elastic:<your-elastic-password>"</span></li>
+                                <li>In the curl section, change <span class="courier-font">"http://localhost:8081/elastic/_bulk", to "https://localhost:9200/elastic/_bulk"</span></li>
```

Comment:
On lines 517 and 514, I think the word "to" should be outside the span. So it's "change ... to ..."

Reply:
done.